src/store/ChartState.ts

```diff
@@ -453,251 +374,62 @@ class ChartState {
             (this.chartType && this.mainStore.chartType.isTypeCandle(this.chartType));
 
         if (this.context && isTimeUnitSecond && isChartTypeCandle) {
-            this.setChartType('mountain'); // if granularity is zero, set the chartType to 'mountain'
+            this.setChartType('line'); // if granularity is zero, set the chartType to 'mountain'
```

Comment:
```suggestion
            this.setChartType('line'); // if granularity is zero, set the chartType to 'line'
```
@balakrishna-deriv nit-pick🙂